src/Acclaim/Provider.php

```diff
@@ -59,13 +59,5 @@ protected function mapUserToObject(array $user)
         ]);
     }
 
-    /**
-     * {@inheritdoc}
-     */
-    protected function getTokenFields($code)
-    {
-        return array_merge(parent::getTokenFields($code), [
-            'grant_type' => 'authorization_code',
-        ]);
-    }
+
```

Comment:
Double newline at end of every file?

Reply:
Fixed, see https://github.com/SocialiteProviders/Providers/pull/1072/commits/cb755ebae4b2aaf564782a25ddeeaa288328b799